modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -4937,6 +4941,17 @@ else if (spiState == CONNECTING)
                 sendMessageAcrossRing(msg);
         }
 
+        /**
+         * @param secCtx Sec context.
```

Comment:
Please use full word in parameter description.

Reply:
Fixed.